src/test/java/org/folio/rest/impl/SynchronizationAPITests.java

```diff
@@ -287,6 +326,13 @@ protected void runSynchronization() {
       .statusCode(202);
   }
 
+  protected void checkSyncJob(String syncJobId) {
+    Awaitility.await()
+      .atMost(30, SECONDS)
+      .until(() -> waitFor(synchronizationJobRepository.get(syncJobId))
+        .orElse(null), is(synchronizationJobMatcher(JOB_STATUS_DONE, 0, 0, 0, 0)));
```

Comment:
do not wrap `synchronizationJobMatcher` in `is`

Reply:
fixed